kong/plugins/request-transformer/access.lua

```diff
@@ -439,31 +448,30 @@ local function transform_body(conf)
   end
 
   if is_body_transformed then
-    req_set_body_data(body)
-    req_set_header(CONTENT_LENGTH, #body)
+    set_raw_body(body)
+    set_header(CONTENT_LENGTH, #body)
   end
 end
 
 local function transform_method(conf)
   if conf.http_method then
-    req_set_method(ngx["HTTP_" .. conf.http_method:upper()])
+    set_method(conf.http_method:upper())
     if conf.http_method == "GET" or conf.http_method == "HEAD" or conf.http_method == "TRACE" then
-      local content_type_value = req_get_headers()[CONTENT_TYPE]
+      local content_type_value = get_headers()[CONTENT_TYPE]
       local content_type = get_content_type(content_type_value)
       if content_type == ENCODED then
         -- Also put the body into querystring
 
         -- Read the body
-        req_read_body()
-        local body = req_get_body_data()
+        local body = get_raw_body()
         local parameters = decode_args(body)
 
         -- Append to querystring
-        local querystring = req_get_uri_args()
+        local querystring = get_uri_args()
         for name, value in pairs(parameters) do
           querystring[name] = value
```

Comment:
Do we want `append` or `add or replace`? This seems to replace existing. Also PDK has `add_header` for `append`?